### ISSUE TITLE
Handle migration of sprites from trial version to full version

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#533]
There should be a way to copy and paste or transfer documents from the trial version to the full version so the document can be saved.

**Comment by dacap** [comment]
Related to #172 and the copypaste library. We need to use custom clipboard formats to fix this issue.

**Comment by chavelka** [comment]
I have just downloaded v1.1 and there is not way to migrate a sprite from the trial to the purchased version in OSX. Copy and past does not work. I can run both simultaneously but there is no way to save the file , cut copy etc.

**Comment by dacap** [comment]
For Linux clipboard support see #1100

**Comment by dacap** [comment]
I'm reopening this issue because we should add support for custom clipboard format to copy and paste layers between different instances (e.g. trial -> full version).

**Comment by alinnert** [comment]
I don't know if you've thought about it, but there's a similar approach to this which is used by FL Studio. It does let you save your work in the trail version, but it doesn't let you re-open your saved files - you can open demo files though.

That way you can simply save your work in the trail and open it in the full version. Maybe that's easier than dealing with the problems the clipboard seems to cause you.

**Comment by gfoal** [comment]
> I'm reopening this issue because we should add support for custom clipboard format to copy and paste layers between different instances (e.g. trial -> full version).

Hi, I just bought Aseprite after using trial version and I can't copy-paste layers from trial into full. Is there anything that can be done?

**Comment by dacap** [comment]
@gfoal you should be able to copy and paste the images (not the full layer), check https://community.aseprite.org/t/saving-on-the-trial-version/5638/2

**Comment by gfoal** [comment]
@dacap yeah, it didn't work if the layer was selected (yellow border) when I was copying. works when it's not selected
<img width="237" alt="image" src="https://user-images.githubusercontent.com/7746914/98040017-4c25c180-1e28-11eb-9d9b-a5ec5ac04afb.png">

thanks for the quick reply 🙇‍♂️
